README.md

```diff
@@ -77,11 +77,13 @@ A list of available commands is accessible by running `codeclimate` or
 $ codeclimate help
 
 Available commands:
-    analyze [-f format] [-e engine(:channel)] [--dev] [path]
+    analyze [-f format] [-e engine[:channel]] [path]
     console
     engines:install
     engines:list
-    help
+    help [command]
+    prepare [--allow-internal-ips]
+    test <engine_name>
```

Comment:
This is not really a supported command, despite the code still existing for it. We don't recommend engine authors use it, so I think we'd like to keep it out of the README.

Reply:
what isn't? `test` ?

Reply:
Yes. `test` is not encouraged to be used, and we'd prefer to leave it undocumented.

Reply:
codeclimate prepare